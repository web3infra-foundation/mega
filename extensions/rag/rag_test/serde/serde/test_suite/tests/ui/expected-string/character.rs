use serde_derive::Serialize;

#[derive(Serialize)]
struct S {
    #[serde(rename = 'a')]
    character: (),
}

fn main() {}
