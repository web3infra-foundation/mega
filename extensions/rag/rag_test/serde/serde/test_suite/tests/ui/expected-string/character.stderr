error: expected serde rename attribute to be a string: `rename = "..."`
 --> tests/ui/expected-string/character.rs:5:22
  |
5 |     #[serde(rename = 'a')]
  |                      ^^^
